freshmaker/pyxis.py

```diff
@@ -0,0 +1,246 @@
+
+        try:
+            response_text = response.json()
+        except ValueError:
+            response_text = response.text
+
+        raise PyxisRequestError(response.status_code, response_text)
+
+    def _pagination(self, entity, params):
+        """
+        Process all pages in Pyxis
+
+        :param str entity: what data/entity to request from Pyxis
+        :param dict params: parameters to add to GET request
+        :return: list of all 'data' fields from responses from Pyxis
+        :rtype: list
+        """
+        local_params = {"page_size": "100"}
+        local_params.update(params)
+        ret = []
+        page = 0
+        while True:
+            local_params["page"] = page
+            response_data = self._make_request(entity, params=local_params)
+            # When the page after the actual last page is reached, data will be an empty list
+            if not response_data.get('data'):
+                break
+            ret.extend(response_data['data'])
+            page += 1
+
+        return ret
+
+    def get_operator_indices(self, params=None):
+        """
+        Get all index images for organization(s)(configurable) from Pyxis
+
+        :param dict params: parameters to add to GET request
+        """
+        if params is None:
+            request_params = {}
+        else:
+            request_params = copy.deepcopy(params)
+        organization = conf.pyxis_index_image_organization
+        if organization:
+            request_params["filter"] = "organization==" + organization
+
+        return self._pagination("operators/indices", request_params)
+
+    def _get_bundles_per_index_image(self, index_images, params=None):
+        """
+        Get bundle images for all index images
+
+        :param list index_images: list of index images
+        :param dict params: parameters to add to GET request
+        :return: bundle images per index image
+        :rtype: dict
+        """
+        if params is None:
+            request_params = {}
+        else:
+            request_params = copy.deepcopy(params)
+        bundles_per_index_image = {}
+        # we need 'bundle_path_digest' to find ContainerImage of that bundle
+        include_fields = ['data.channel_name', 'data.version',
+                          'data.related_images', 'data.bundle_path_digest',
+                          'data.bundle_path']
+        request_params['include'] = ','.join(include_fields)
+        for index_image in index_images:
+            path = index_image.get('path', '')
+            if not path:
+                continue
+            request_params['filter'] = f"source_index_container_path=={path}"
+            bundles_per_index_image[path] = self._pagination('operators/bundles',
+                                                             request_params)
+
+        return bundles_per_index_image
+
+    def get_latest_bundles(self, index_images, params={}):
```

Comment:
Looks like you missed this one. 
```suggestion
    def get_latest_bundles(self, index_images, params=None):
```